Libraries/D3D12RaytracingFallback/src/RayTracingHlslCompat.h

```diff
@@ -402,6 +402,23 @@ uint GetNumInternalNodes(uint numLeaves)
     return numLeaves - 1;
 }
 
+inline
+uint GetOffsetFromSortedIndicesToAABBParents(uint numPrimitives) {
+    return 4 /* sizeof(UINT) */ * numPrimitives;
```

Comment:
Use SizeOfUINT32 instead. You'll need to move the #define for that from RaytracingHelper.hlsli to HlslCompat.h for this to work, but I'm a big fan of having self-documenting code over comments